neo/Consensus/ConsensusService.cs

```diff
@@ -299,20 +303,30 @@ private void OnRecoveryMessageReceived(ConsensusPayload payload, RecoveryMessage
                     {
                         ConsensusPayload prepareRequestPayload = message.GetPrepareRequestPayload(context, payload);
                         if (prepareRequestPayload != null)
-                            ReverifyAndProcessPayload(prepareRequestPayload);
+                        {
+                            totalPrepReq++;
```

Comment:
Is there any chance for `totalPrepReq` to be greater then `1`?

Reply:
No but it looks nicer for the output to be uniform.